### PR TITLE
[docs] Update README.android

### PR DESCRIPTION
**Pull request by santaSJ**
Update README.android

## Description

Update README.android to use cmakebuildsys
Add libcurl4-openssl-dev to the list of required packages
## Motivation and Context

Updated README will help people build / contribute to Kodi faster
## How Has This Been Tested?

Tested on a fresh installation of Ubuntu 16.04  64Bit
## Types of change

Only Documentation was changed
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] My code follows the [Code guidelines](https://codedocs.xyz/xbmc/xbmc/code_guidelines.html) of this project 
- [ ] My change requires a change to the documentation, either Doxygen or wiki
- [ ] I have updated the documentation accordingly
- [ ] I have read the [CONTRIBUTING](https://github.com/xbmc/xbmc/blob/master/CONTRIBUTING.md) document
- [ ] I have added tests to cover my change
- [ ] All new and existing tests passed

### CODE REVIEW

File: docs/README.android [hunk 1]
```diff
@@ -29,7 +29,7 @@ These are the minimum packages necessary for building Kodi.
 Non-Ubuntu users will need to get the equivalents.
 
     $ sudo apt-get install build-essential default-jdk git curl autoconf \
-      unzip zip zlib1g-dev gawk gperf cmake
+      unzip zip zlib1g-dev gawk gperf cmake libcurl4-openssl-dev
```
Comment (wsnipex) [inline docs/README.android]: why is that needed? We build openssl in depends

Comment (mrdominuzq) [inline docs/README.android]: http://forum.kodi.tv/showthread.php?tid=216322 :)
